catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -562,6 +557,18 @@ private SourceResponse cloneResponse(SourceResponse sourceResponse) {
           sourceResponse.getHits(),
           sourceResponse.getProperties());
     }
+
+    private Function<Result, Result> cloneResult =
```

Comment:
✏️ This is fine. Could also just declare a method, `private Result cloneResult(Result)`.